### ISSUE TITLE
No command available for SF4

### ISSUE THREAD

**Issue by jewome62** [avanzu/AdminThemeBundle#213]
* Install dev-master on symfony 4
  * try one of command avanzu                                                         
  * Message is display "There are no commands defined in the "avanzu:admin" namespace. "
  * See #126

### Debug info

| Component  | Version |
| ------------- | ------------- |
| Symfony version  | 4.0.3  |
| AdminThemeBundle  | dev-master  |

**Comment by shakaran** [comment]
@jewome62 this bundle still doesn't have support for Symfony 4.x and probably will take some time until get it (since the focus now is stabilize the bugs related to 2.x version of this bundle and release as stable). Pull request are welcome if you desire speed up the progress.

**Comment by lxxps** [comment]
Quick workaround, in config/services.yaml file

```
    # in case any automatic discovery fails, register all commands in one declaration :)
    Avanzu\AdminThemeBundle\Command\:
        resource: "../vendor/avanzu/admin-theme-bundle/Command/*"
        tags: [ "console.command" ]
```

**Comment by kevinpapst** [comment]
This would be fixed with #216

**Comment by shakaran** [comment]
@jewome62 @lxxps master branch should be now more compatible with SF4. Test if you want. The commands should be now available.
